### PR TITLE
feat(offcanvas): Added Offcanvas submenu functionality.

### PR DESCRIPTION
**Pull request by tszekely**
I am not sure about the unit tests, as this is the first time I ever do it, but I hope it's a cool addition overall.
Also, I've bumped the Foundation CSS version to 5.4.6 (for the submenu CSS).

### CODE REVIEW

**Comment by justageek** [comment]
Hi, is this ready to merge, are there still merge conflicts, I grabbed only the code in off-canvas.js and just pasted into the current downloaded code and it appears to work.

File: src/offcanvas/offcanvas.js [hunk 1]
```diff
@@ -73,8 +73,17 @@ angular.module("mm.foundation.offcanvas", [])
             require: '^offCanvasWrap',
             restrict: 'C',
             link: function ($scope, element, attrs, offCanvasWrap) {
-                element.on('click', function () {
-                    offCanvasWrap.hide();
+                element.find('li').on('click', function (e) {
```
Comment (janschaefer) [inline src/offcanvas/offcanvas.js]: the solution has another drawback. When dynamically creating the menu, this solution does not work.

**Comment by NeXTs** [comment]
Any updates?

**Comment by Tonio31** [comment]
This change is exactly what I need, does anybody plan to approve this pull request. It is really a shame that this funcitonality doesn't work with Angularjs (I tested the pull request manually in my code and it works)

**Pull request closed without merging**